HTAN.model.jsonld

```diff
@@ -52090,6 +52282,23 @@
             "sms:required": "sms:false",
             "sms:validationRules": []
         },
+        {
+            "@id": "bts:MSAssayCategory",
+            "@type": "rdfs:Class",
+            "rdfs:comment": "TBD",
```

Comment:
MS Assay Category needs to be given a description

Reply:
OK. there were two features that needed descriptions.